src/NUnitFramework/framework/Attributes/SingleThreadedAttribute.cs

```diff
@@ -28,11 +28,11 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// SingleThreadedAttribute applies to a test fixture and indicates
-    /// that all the child tests must be run on the same thread as the
-    /// OneTimeSetUp and OneTimeTearDown. It sets a flag in the
-    /// TestExecutionContext and forces all tests to be run sequentially
-    /// on the current thread. Any ParallelScope setting is ignored.
+    /// Marks a test fixture as requiring all child tests must be run on the 
+    /// same thread as the OneTimeSetUp and OneTimeTearDown. A flag in the
+    /// <see cref="TestExecutionContext"/> is set and forces all tests to be run 
+    /// sequentially on the current thread. Any <see cref="ParallelScope"/> 
+    /// setting is ignored.
```

Comment:
"A flag in the TestExecutionContext is set and forces..." doesn't sound right. Should this be "which forces" now, instead?